Ethane

```diff
@@ -0,0 +1,4 @@
+methyl = [0,2,3,4]
+list = [methyl, methyl]
+
```

Comment:
Even though the methyls have symmetry you need to specify them independently because PySCF and `guess_driver` don't know about that symmetry, *i.e.* one of the inner lists needs to be `[1, 5, 6, 7]`.